gsoft-inc/sg-orbit
# Fix Storybook for Edge

Issue: https://github.com/gsoft-inc/sg-orbit/issues/69

## What I did

Added SB existing Babel options to babel config for MDX files

## How to test

Open storybook in Edge

- Is this testable with Jest or Chromatic screenshots? No
- Does this need an update to the documentation? No

If your answer is yes to any of these, please make sure to include it in your PR.
